src/core/Microsoft.PowerToys.Settings.UI/Views/PowerPreviewPage.xaml

```diff
@@ -43,25 +48,41 @@
 
         <StackPanel Orientation="Vertical" 
                     x:Name="PowerPreviewView">
+            <TextBlock x:Uid="FileExplorerPreview_RunAsAdminRequired"
+                       Foreground="{ThemeResource SystemControlErrorTextForegroundBrush}"
+                       Visibility="{Binding Mode=OneWay, Path=IsElevated, Converter={StaticResource BoolToVisibilityConverter}}"
+                       Margin="{StaticResource SmallBottomMargin}"/>
+
+            <TextBlock x:Uid="FileExplorerPreview_AffectsAllUsers"
```

Comment:
Because it is a hint/warning I would paint it red too:

```suggestion
            <TextBlock x:Uid="FileExplorerPreview_AffectsAllUsers"
            Foreground="{ThemeResource SystemControlErrorTextForegroundBrush}"
```